### PR TITLE
Update submodule to latest, Fix CI build

### PR DESCRIPTION
**Pull request by iambmelt**
Work in progress.

Goals of this PR:
- Fix unit tests to use `androidx testrunner`
- Update submodule to `common/dev` latest

### CODE REVIEW

File: adal/src/androidTest/java/com/microsoft/aad/adal/AuthenticationContextTest.java [hunk 1]
```diff
@@ -33,10 +33,12 @@
 import android.content.pm.Signature;
 import android.os.Build;
 import android.os.Bundle;
+import android.util.Base64;
+
 import androidx.test.InstrumentationRegistry;
```
Comment (rpdome) [inline adal/src/androidTest/java/com/microsoft/aad/adal/AuthenticationContextTest.java]: given that we seems to be using androidx.test.platform.app.InstrumentationRegistry, can we remove this for good?

Comment (rpdome) [reply adal/src/androidTest/java/com/microsoft/aad/adal/AuthenticationContextTest.java]: ps. I have zero idea why there are InstrumentationRegistry in two packages.

Comment (iambmelt) [reply adal/src/androidTest/java/com/microsoft/aad/adal/AuthenticationContextTest.java]: Yeah, missed this import - I'll run the tool

File: gradle/versions.gradle [hunk 2]
```diff
@@ -8,7 +8,7 @@ ext {
     buildToolsVersion = "28.0.3"
 
     // Plugins
-    gradleVersion = '3.2.1'
+    gradleVersion = '3.3.1'
```
Comment (rpdome) [inline gradle/versions.gradle]: Was going to do this. Nice catch. Where can I add thumbs up emoji.

File: adal/src/androidTest/java/com/microsoft/aad/adal/AuthenticationContextTest.java (hunk 1 above)
Comment (iambmelt) [reply adal/src/androidTest/java/com/microsoft/aad/adal/AuthenticationContextTest.java]: Resolved in https://github.com/AzureAD/azure-activedirectory-library-for-android/pull/1449

File: gradle/versions.gradle (hunk 2 above)
Comment (iambmelt) [reply gradle/versions.gradle]: Right here! 💯

**Pull request merged**